src/commands/CommandHandler.ts

```diff
@@ -214,6 +217,19 @@ export async function handleCommand(roomId: string, event: { content: { body: st
                 "!mjolnir resolve <room alias>                                       - Resolves a room alias to a room ID\n" +
                 "!mjolnir shutdown room <room alias/ID> [message]                    - Uses the bot's account to shut down a room, preventing access to the room on this server\n";
 
+            const labsHtmlPreamble =
+                "<br />" +
+                "⚠️ Note: These commands are experimental and subject to change. Check <code>!mjolnir help</code> for the latest version of these commands.<br />" +
+                "⚠️ Note: <a href='https://github.com/matrix-org/matrix-spec-proposals/pull/4284'>MSC4284</a> commands may require additional configuration before the policy server will operate. Consult the policy server provider's documentation for more information.<br />" +
+                ""; // empty segment to reduce diff noise
+            const labsMenu =
+                "" +
+                "!mjolnir msc4284_set <room alias/ID> <server name>                  - Set the policy server in a specific room. Mjolnir doesn't need to be protecting the room, but needs permission to send state events.\n" +
```

Comment:
In this and below, I am assuming that <server name> is referring to the policy server name? It might be helpful to make that explicit.

Reply:
good call - thanks! I've made the change in https://github.com/matrix-org/mjolnir/pull/605/commits/041d8badcb51a2502f0216bfba8074b8f514ea0b